lib/render/AnnotationRenderer.cpp

```diff
@@ -698,7 +736,7 @@ void AnnotationRenderer::renderText(double text, double coord[], AxisAnnotation
     label.VerticalAlignment = TextLabel::Top;
     label.Padding = fontSize / 4.f;
     label.ForegroundColor = glm::vec4(axisColor[0], axisColor[1], axisColor[2], axisColor[3]);
-    label.BackgroundColor = glm::vec4(backgroundColor[0], backgroundColor[1], backgroundColor[2], backgroundColor[3]);
+    label.BackgroundColor = glm::vec4(backgroundColor[0], backgroundColor[1], backgroundColor[2], 1.);
```

Comment:
When I was experimenting with this, I was unable to find an alpha channel selector in the Mac color-picker.  It's odd, because I believe it was there before we were using the PColorSelector but I could be mistaken.
![Screen Shot 2021-04-30 at 8 11 17 AM](https://user-images.githubusercontent.com/9522770/116707261-b2307300-a98b-11eb-8047-7d981c0d7fa6.png)

Reply:
Ok but the input to render text still allows specifying an opacity. Why would the foreground color be different then?

Reply:
The foreground's default color and opacity is 1,1,1,1.  The background was 0,0,0,0.  Without a way to apply an opacity value, we had permanently transparent backgrounds, so I hard coded the opacity to be 1.

What would you suggest we do here?

Reply:
I did not realize this was done to fix a bug. Nevermind